### DISCUSSION TITLE
Select inside a Dropdown

### DISCUSSION THREAD

**Discussion by cdeutsch** [radix-ui/primitives#1411] [Help] [answered]
I have a UI that has Selects nested inside a Dropdown.

In `@radix-ui/react-dropdown-menu@0.1.6` and `@radix-ui/react-select@0.1.1`, it looks like this works when using a mouse, but when using the keyboard I can't tab to the Select. I'm stuck on the Content div (highlighted in blue)

![image](https://user-images.githubusercontent.com/404765/169410655-12fe0167-fd57-4fe8-8522-229ab97d824e.png)

Is this a bug, or are Selects nested in Dropdowns not supported?

If the latter, is there any plans to support them in the future?

**Comment by cdeutsch** [comment, marked as answer]
After spending some more time with this, switching to `Popover` solved my problem.

I guess `Dropdown` is meant to be limited to the predefined children?

**Reply by benoitgrelard** [reply]
Hey @cdeutsch, it looks like you found the solution yourself. `DropdownMenu` is specifically made for a dropdown menu with items. `Popover` is much more general (basically a positioned/anchored dialog) and looking more like what you were after all along.
